README.md

```diff
@@ -1,44 +1,111 @@
+
+## Download the Cloud in a Box image
+
+> **Warning:** ⚡ When booting from this image, all data on the hard disks will be destroyed without confirmation.
+
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-1.iso> (with /dev/sda)
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-2.iso> (with /dev/nvme0n1)
+
+## Getting started
+
+Download the Cloud in a Box image from the link above.
+
+Use a tool to create a bootable USB stick with the Cloud in a Box image.
+
+Boot from the USB stick.
+
+The instalation will start and take a few minutes. After that the system will shutdown.
+
+Remove the USB stick and restart the system.
+
+The deployment will start. This takes some time and the system will shutdown when the deployment is finished.
+This takes roughly an hour, possibly longer depending on the hardware and internet connection.
+
+System is ready for use, by default DHCP is tried on the first network device.
+
+Login via ssh. Use the user **dragon** with the password **password**.
+
+```bash
+ssh dragon@IpFromYourServer
+```
+
+Copy the `/home/dragon/wireguard-client.conf` file to your device. This is necessary for using the web endpoints on your device.
+Rename the wireguard config file to something like `ciab.conf`.
+> **Note:**
+> Be aware that the name does not contain any special characters!
+
+```bash
+scp dragon@IpFromYourServer:/home/dragon/wireguard-client.conf /home/ubuntu/ciab.conf
+```
+
+Install wireguard on your device, if you have not done this before. For instructions how to do it on your device, please have
+a look on the documentation of your used distribution.
+
+Start the wireguard tunnel.
+
+```bash
+wg-quick up /home/ubuntu/ciab.conf
+```
+
+Now your Cloud in a Box is up and you can reach the most services via the Homer dashboard:
+<https://homer.services.in-a-box.cloud>
+
+If you want to access the services directly please choose the URL from the following list:
+
+| Name              | URL                                           | Username  | Password                                 |
+|-------------------|-----------------------------------------------|-----------|------------------------------------------|
+| ARA               | <https://ara.services.in-a-box.cloud>         | ara       | password                                 |
+| Flower            | <https://flower.services.in-a-box.cloud>      | -         | -                                        |
+| Grafana           | <https://api.in-a-box.cloud:3000>             | admin     | password                                 |
+| Homer             | <https://homer.services.in-a-box.cloud>       | -         | -                                        |
+| Horizon Admin     | <https://api.in-a-box.cloud>                  | admin     | password                                 |
+| Horizon Test-User | <https://api.in-a-box.cloud>                  | test      | test # FIX ME                            |
```

Comment:
test password is fine

Reply:
It was belong to this issue as a reminder to ourselve https://github.com/osism/cloud-in-a-box/issues/17. Since this is closed now we can delete the fix me